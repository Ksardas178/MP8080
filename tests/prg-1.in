LXI SP o202 o000
LXI B o222 o333
PUSH B
POP D
INR E
DCR D
PUSH D
HLT