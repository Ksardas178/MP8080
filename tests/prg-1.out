
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: LXI 6 130 0
00000001: LXI 0 146 219
00000010: PUSH 0
00000011: POP 2
00000100: INR 3
00000101: DCR 2
00000110: PUSH 2
00000111: HLT
